### DISCUSSION TITLE
b-table selected column

### DISCUSSION THREAD

**Discussion by jeffplata** [bootstrap-vue/bootstrap-vue#7189] [General]
in b-table, when the `selectable` option is enabled, a `selected` column is added in column 1. How do I remove it?
![image](https://github.com/bootstrap-vue/bootstrap-vue/assets/29348635/cc78800b-3645-4abc-b2cd-ffaa2be6468a)

**Comment by Hiws** [comment]
You're posting in the wrong repo.

This is the one you want: https://github.com/bootstrap-vue-next/bootstrap-vue-next
also, just update.
